packages/cli/src/lib/webpack/webpack-base-config.js

```diff
@@ -329,7 +321,6 @@ module.exports = function createBaseConfig(config, env) {
 						},
 					},
 				}),
-			isProd && new webpack.LoaderOptionsPlugin({ minimize: true }),
```

Comment:
Super legacy, [exists to smooth over Webpack 1 -> 2 migration woes](https://v4.webpack.js.org/plugins/loader-options-plugin/)

We've just had it sat around forever.